core-war/src/main/webapp/util/javaScript/checkForm.js

```diff
@@ -243,4 +243,25 @@ function checkHour(hour)
 function checkemail(email) {
 	var re = /^(([^<>()[\]\\.,;:\s@\"]+(\.[^<>()[\]\\.,;:\s@\"]+)*)|(\".+\"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
   return email === '' || !re.test(email) ? false : true;
-}
\ No newline at end of file
+}
+
+/**
+ * Verifies some transversal rules about file upload.
+ * @param $fileInput the instance of the input into the DOM.
+ * @returns {Promise<*>}
+ */
+window.verifyFileInput = function($fileInput){
+    const [mediaFile] = $fileInput.files;
```

Comment:
Are you sure there it will be always a single file? In my advice , to secure this method, it should browse the array and for each File object, performs the verification.

Reply:
Actually, this method has been created for one file verification. We have to talk to Yohann to discuss about this question.

Reply:
So, to ensure this choice, the method should have one argument representing a single file and not an HTML element from which an array of files can be fetched. In fact, the better option in my advice is to modify the signature of the method to accept a plain js object and not an HTML element: either an array of File instances or a single File object. By doing so you ensure the caller what it is actually expected by the method (on what it works)

Reply:
Finally,  the first solution was taken and its use in `editMedia.tag` modified as 
```javascript
errorPromises.push(verifyFileInput($media.files.at(1)));
```

Reply:
<!-- Issues highlight specific problems with the subject under review. These problems can be user-facing or behind the scenes. It is strongly recommended to pair this comment with a suggestion. If you are not sure if a problem exists or not, consider leaving a question.  -->
**issue:** ‏```$media.files.at(0)```
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution.  -->
**question:** ‏ what is happening if the user validates while no file has been indicated into input? (create/modify cases)
<!-- Suggestions propose improvements to the current subject. It's important to be explicit and clear on what is being suggested and why it is an improvement. Consider using patches and the blocking or non-blocking decorations to further communicate your intent.  -->
**suggestion:** ‏ the code on my previous comment

Reply:
**Response:** undefined is returned by `media.files.at(0)`. And, in my version, as the if-statement to check mediaFile is defined and not null is kept, then nothing happens as it was in the original version

Reply:
Nevertheless, as you're insisting about this, I will use your proposition